### DISCUSSION TITLE
Quantification strategy for Silac

### DISCUSSION THREAD

**Discussion by Giu-F** [vdemichev/DiaNN#975] [Q&A]
Hi, which quantification setting from the QuantUMS paper/Diann 1.8.2 is recommended to use for Silac? High precision or high accuracy?
In this version, does it make sense anymore to use Ms1 quantification (Ms1 aerea column) - like it was done in the PlexDia paper? Since Ms1 is also used to calculate the Precursor quantity column, I guess this is the only quantity that makes sense to use, right?

**Comment by vdemichev** [comment]
In beta 27 QuantUMS is not yet optimised for multiplexing, so please use the legacy quant method. Or Ms1.Normalised.

**Reply by Giu-F** [reply]
When using Diann 1.8.1 and the quantity precursor translated for quantification, is there an optimal quantification strategy to choose?

**Reply by vdemichev** [reply]
The default one.

**Reply by Giu-F** [reply]
Hi, in Diann 1.9 which QuantUMS setting is recommended to use for Silac? High precision or high accuracy?

**Reply by vdemichev** [reply]
Either, high-precision is currently default and is therefore recommended for most scenarious
